### PR TITLE
Add a kubekey internal loadbalancer usage doc in zh docs

### PR DESCRIPTION
**Pull request by 24sama**
### What does this PR do?
Add a kubekey internal loadbalancer usage doc in zh docs. 

### Why we need this PR?
I think this doc can help users easy to use kubekey to create a ha cluster.

/cc @FeynmanZhou 
/cc @Patrick-LuoYu

### CODE REVIEW

File: content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+title: "使用KubeKey创建高可用集群"
```
Comment (FeynmanZhou) [inline content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md:2]: 使用 KubeKey 内置 HAproxy 创建高可用集群

File: content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+keywords: 'KubeSphere, Kubernetes, KubeKey, HA, 高可用, 安装, 配置'
```
Comment (FeynmanZhou) [inline content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md:3]: Please list only 5 keywords or less.

File: content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+description: '如何配置一个高可用 Kubernetes 集群。'
+linkTitle: "使用KubeKey创建高可用集群"
```
Comment (FeynmanZhou) [inline content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md:5]: 使用 KubeKey 内置 HAproxy 创建高可用集群

File: content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
+---
+title: "使用KubeKey创建高可用集群"
+keywords: 'KubeSphere, Kubernetes, KubeKey, HA, 高可用, 安装, 配置'
+description: '如何配置一个高可用 Kubernetes 集群。'
```
Comment (FeynmanZhou) [inline content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md:4]: 如何使用 KubeKey 内置的 HAproxy 安装一个高可用的 KubeSphere 与 Kubernetes 集群。

File: content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+linkTitle: "使用KubeKey创建高可用集群"
+weight: 3150
+---
+
+[KubeKey](https://github.com/kubesphere/kubekey) 作为一种集群安装工具，从版本 v1.2.0 开始，提供了内置高可用模式，支持一键部署高可用集群环境。KubeKey 的高可用模式实现方式称作本地负载均衡模式。具体表现为 KubeKey 会在每一个工作节点上部署一个负载均衡器（haproxy），所有主节点的 Kubernetes 组件连接其本地的 kube-apiserver ，而所有工作节点的 Kubernetes 组件通过由 KubeKey 部署的负载均衡器反向代理到多个主节点的 kube-apiserver 。这种模式相较于专用到负载均衡器来说效率有所降低，因为会引入额外的健康检查机制，但是如果当前环境无法提供外部负载均衡器或者虚拟 IP（VIP）时这将是一种更实用、更有效、更方便的高可用部署模式。
```
Comment (FeynmanZhou) [inline content/zh/docs/installing-on-linux/high-availability-configurations/internal-ha-configuration.md:9]: haproxy => HAproxy

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
Please see my comments above, thanks!

**Comment by 24sama** [comment]
Hi @FeynmanZhou 
Thanks for the review. I have modified it.

**Pull request merged**